batch/airflow/airflow-sa.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: airflow
+  namespace: airflow
+  annotations:
+    iam.gke.io/gcp-service-account: airflow-dags-reader@lustrous-baton-363720.iam.gserviceaccount.com
```

Comment:
Is this hardcoded to somewhere? (`lustrous-baton-363720`)

Reply:
It's leftover from testing. I'll fix it.